geodata/admin.py

```diff
@@ -15,61 +20,74 @@
 )
 
 
-@admin.register(Dataset)
-class DatasetAdmin(OSMGeoAdmin):
+@admin.register(ImageSet)
+class ImageSetAdmin(OSMGeoAdmin):
     list_display = (
         'id',
         'name',
     )
 
 
+@admin.register(ImageEntry)
+class ImageEntryAdmin(OSMGeoAdmin):
+    list_display = (
+        '__str__',
+        'modified',
+    )
+    readonly_fields = (
+        'number_of_bands',
+        'image_file',
+        'height',
+        'width',
+        'driver',
+        'metadata',
+        'modified',
+        'created',
+    )  # 'thumbnail')
+    list_filter = ('instrumentation', 'number_of_bands', 'driver')
+
+
 @admin.register(RasterEntry)
 class RasterEntryAdmin(OSMGeoAdmin):
     list_display = (
         '__str__',
         'modified',
     )
     readonly_fields = (
-        'number_of_bands',
-        # NOTE: we do not want users to edit these, but the geodjango interface is really nice
-        # 'footprint',
-        # 'outline',
-        'raster_file',
+        'footprint',
```

Comment:
We can still show a map.  Set `modifiable = False` as a class property and the maps are read-only, then remove the `footprint` and `outline` fields from the `readonly_fields` dictionary.

Reply:
Done. The only caveat that I see to this is that when creating a RasterEntry, the map is shown for the user to select these polygons. I had to set `blank=True` to those fields to allow the user to not fill those out since they are auto-populated